## joint_names
joint_names: [I1_Joint, T2_Joint, T3_Joint, i4_Joint, t5_Joint]

## control_mode_adapter
max_command_silence: 0.5

## joint_state_controller
joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

## joint trajectory controller
joint_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints:
     - I1_Joint
     - T2_Joint
     - T3_Joint
     - i4_Joint
     - t5_Joint
    
#  constraints:
#      goal_time: 0.25
#      stopped_velocity_tolerance: 0.05
#      I1_Joint: {trajectory: 0.1, goal: 0.1}

  stop_trajectory_duration: 0.25
  state_publish_rate:  25
  action_monitor_rate: 10
  required_drive_mode: 7


## position controller
joint_group_position_controller:
  type: position_controllers/JointGroupPositionController
  joints:
     - I1_Joint
     - T2_Joint
     - T3_Joint
     - i4_Joint
     - t5_Joint
  required_drive_mode: 1
#  pid:
#    p: 100.0
#    d: 10.0  
    
I1_Joint_position_controller:
  type: position_controllers/JointPositionController
  joint: I1_Joint
  required_drive_mode: 1

  
## velocity controller
joint_group_velocity_controller:
  type: velocity_controllers/JointGroupVelocityController
  joints:
     - I1_Joint
  required_drive_mode: 3

I_Joint_velocity_controller:
  type: velocity_controllers/JointVelocityController
  joint: I1_Joint
  required_drive_mode: 3



